Sources/Paywalls/PaywallData.swift

```diff
@@ -37,6 +68,13 @@ public struct PaywallData {
         set { self._revision = newValue }
     }
 
+    /// The storefront country IDs that should not display cents in prices.
+    public var zeroDecimalPlaceCountries: ZeroDecimalPlaceCountries {
```

Comment:
Need to add to API tests?

Reply:
[We do for PaywallData](https://github.com/RevenueCat/purchases-ios/blob/main/Tests/APITesters/AllAPITests/SwiftAPITester/PaywallAPI.swift)!